homeassistant/components/azure_servicebus/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "azure_servicebus",
```

Comment:
Should we name it `azure_service_bus`, as it's three words?

Reply:
Tried to research what the correct naming is, with or without separation in `service bus`, but it seems even microsoft isn't consistent on this. Naming in text seems to be two words, but when they use it in libraries and code patterns they use one word. 
If three words in the integration name is more suited for HA, I will of course change it!

Reply:
On Azure website they write the name in three words.
https://docs.microsoft.com/en-us/azure/service-bus-messaging/

And looking at eg `ServiceBusClient` class it looks like the intent is to separate service and bus since Bus is capitalized in the PascalCased name.

I suggest we go with `azure_service_bus` as integration name.

Reply:
👍 Will update both PR's tomorrow